test/test-applications/integrations/LogsInjection.Log4Net/LogsInjection.Log4Net.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">2.0.12</ApiVersion>
+    <DefineConstants Condition="'$(ApiVersion)'&gt;='2.0.5'">$(DefineConstants);LOG4NET_2_0_5</DefineConstants>
+
+    <!-- Required to build multiple projects with the same Configuration|Platform -->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+
+    <ExcludeManagedProfiler>true</ExcludeManagedProfiler>
+    <ExcludeNativeProfiler>true</ExcludeNativeProfiler>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="log4net" Version="$(ApiVersion)" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(ApiVersion)' &gt;= '2.0.5'">
+    <PackageReference Include="log4net.Ext.Json" Version="1.2.15.14586" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\dependency-libs\LogsInjectionHelper\LogsInjectionHelper.csproj" />
+    <ProjectReference Include="..\dependency-libs\PluginApplication\PluginApplication.csproj" />
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
Do we need/want this reference? It's currently referenced transitively, and isn't used directly by this app?

Reply:
Fixed in a1c64bfee75f3e355c2262dc33fc5dab3846e54e